packages/es-components/src/components/containers/tabPanels/Tab.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import defaultTheme from 'es-components-via-theme';
+
+/* eslint-disable no-confusing-arrow */
+const TabButton = styled.button`
+  background-color: ${props =>
+    props.selected ? props.theme.colors.white : props.theme.colors.gray4};
```

Comment:
Use gray2 here and throughout for the background hover color. The border can still use gray4.